WordPress/UITestsFoundation/Screens/PagesScreen.swift

```diff
@@ -0,0 +1,32 @@
+import ScreenObject
+import XCTest
+
+public class PagesScreen: ScreenObject {
+    public let tabBar: TabNavComponent
+
+    let pagesTableGetter: (XCUIApplication) -> XCUIElement = {
+        $0.tables["PagesTable"]
+    }
+
+    var pagesTable: XCUIElement { pagesTableGetter(app) }
+
+    init(app: XCUIApplication = XCUIApplication()) throws {
+        tabBar = try TabNavComponent()
+
+        try super.init(
+            expectedElementGetters: [ pagesTableGetter ],
+            app: app,
+            waitTimeout: 7
+        )
+    }
+
+    public static func isLoaded() -> Bool {
+        (try? PagesScreen().isLoaded) ?? false
+    }
+
+    @discardableResult
+    public func verifyPagesScreenLoaded() -> Self {
+        XCTAssertTrue(PagesScreen.isLoaded(), "\"Pages\" screen isn't loaded.")
```

Comment:
depending if this test will be extended again or not, i think we could end up with a false positive if we're only checking for the `PagesTable` table value, i tested with an empty table and saw that this returned 
`Table, 0x7ff76e80c5c0, {{0.0, 137.0}, {390.0, 624.0}}, identifier: 'PagesTable', label: 'Empty list'`

if we're expecting the existing pages to also be listed, we should add additional assertions to validate them so we don't get a passing test when an empty page view is returned.

Reply:
Since I was adding this test as a part of "Pages and Activity Log cards" project, I did not test the "Pages List" screen content, because the test is about card content and working navigation to the Pages List screen. The Pages List screen itself exists for a long time, so automating the check for it would be out of scope.

The same approach is used in `testFreeToPaidCardNavigation`, where the test checks that the `Domains` screen is loaded, but does not check for actual domains suggestions. Let me know if this makes sense, though.